packages/shell-api/src/database.ts

```diff
@@ -780,7 +784,7 @@ export default class Database extends ShellApiClass {
   @returnsPromise
   async enableFreeMonitoring(): Promise<any> {
     this._emitDatabaseApiCall('enableFreeMonitoring', {});
-    const isMaster = await this._mongo._serviceProvider.runCommand(this._name, { isMaster: 1 });
+    const isMaster: any = await this._mongo._serviceProvider.runCommand(this._name, { isMaster: 1 });
```

Comment:
I don't know if it applies here, however I think for most of the results coming from the service provider / server a decent compromise between perfectly specifying the shape of the objects and `any` could be something like this:

```
type ServerCommandResult = Record<string, any>;
type IsMasterResult = ServerCommandResult & {
  isMaster: boolean
};

const res1: IsMasterResult = {
  isMaster: true,
  someOtherProp: 123
};

const res2: IsMasterResult = { // Property 'isMaster' is missing ...
  someOtherProp: 123
};
``` 

I'd love to see if the server actually has some IDL files we could steal those interfaces from.